### ISSUE TITLE
loadingTemplate and layoutTemplate do not work together

### ISSUE THREAD

**Issue by WayneUong** [iron-meteor/iron-router#1521]
```
Router.route('/post', {
    name: 'postRoute',
    loadingTemplate: 'loadingLayout',
    layoutTemplate: 'mainLayout',
    waitOn: function () {
        return Meteor.subscribe('waitForAWhile')
    },
    action: function () {
        console.log('done!');
    }
});
```

Above code will not render loadingLayout while the subscription is going on

**Comment by nemq** [comment]
Hey, do you use  {{>yield}} in your layout template? I run similar setup, just loading template is set globally for all routes and it works fine.

**Comment by WayneUong** [comment]
I think loadingTemplate is set as a default template to {{> yield}}, and layoutTemplate is a default layout set to `<body>`. And I set a name for every yield. That's probably why it's not working. Are these assumptions correct?

**Comment by nemq** [comment]
Yes, i did same mistake. You must have bare {{> yield}} in template for loading to work.
